loadtest.py

```diff
@@ -1,43 +1,93 @@
 import os
-import uuid
 
-from ailoads.fmwk import scenario, requests
+from ailoads.fmwk import scenario
 
-from connection_handler import ConnectionHandler
+import utils
 
 URL_SERVER = os.getenv('URL_SERVER',
-                       'https://antenna.stage.mozaws.net')
-DEBUG = True
+                       'http://antenna.dev.mozaws.net/submit')
```

Comment:
This needs to be `https`. I'm pretty sure that's causing your HTTP 405 problems.